Makefile

```diff
@@ -279,10 +281,23 @@ ${SNITCH_INSTALL_DIR}: ${TOOLCHAIN_DIR}/snitch_cluster
 	TMPDIR=tmp pip install -r python-requirements.txt && rm -rf tmp && \
 	bender vendor init && \
 	cd ${SNITCH_INSTALL_DIR}/target/snitch_cluster && \
-	make sw/runtime/banshee sw/math
+	make sw/runtime/banshee sw/runtime/rtl sw/math
```

Comment:
This change is not mentioned in the PR description. Could you please add it and give the motivation.